### PR TITLE
added position tracker app

### PR DESCRIPTION
**Pull request by Sloth-Panda**
Corresponds to issue -> #212  @Tejas1510

### CODE REVIEW

**Comment by Tejas1510** [comment]
@Sloth-Panda I have always said to make a good Readme File
Add images and follow the proper format provided
Else the code is good to merge

**Comment by Sloth-Panda** [comment]
Okay

File: Javascript/Geo-location/index.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    <p id="demo"></p>
+
+
+    <div id="mapholder"></div>
+
+    <script src="main.js"></script>
+</body>
+
+</html>
```
Comment (atarax665) [inline Javascript/Geo-location/index.html:25]: Add new line at the EOF.

File: Javascript/Geo-location/main.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            x.innerHTML = "Location information is unavailable."
+            break;
+        case error.TIMEOUT:
+            x.innerHTML = "The request to get user location timed out."
+            break;
+        case error.UNKNOWN_ERROR:
+            x.innerHTML = "An unknown error occurred."
+            break;
+    }
+}
```
Comment (atarax665) [inline Javascript/Geo-location/main.js:37]: Add new line at the EOF.

File: Javascript/Geo-location/style.css [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+button{
+    margin-left: 45%;
+    text-align: center;
+    text-transform: uppercase;
+    font-size: 2rem;
+}
+
+#demo{
+    font-size: 1.8rem;
+}
```
Comment (atarax665) [inline Javascript/Geo-location/style.css:15]: Add new line at the EOF.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**